src/textual/widgets/_toggle_button.py

```diff
@@ -161,6 +161,11 @@ def label(self) -> Text:
         """The label associated with the button."""
         return self._label
 
+    @label.setter
+    def label(self, label: TextType) -> None:
+        self._label = Text.from_markup(label) if isinstance(label, str) else label
```

Comment:
The constructor also takes just the first line. Should we not do that here?

Reply:
Excellent catch!

Reply:
Done.